Source/WebInspectorUI/UserInterface/Views/ComputedStyleDetailsPanel.css

```diff
@@ -86,4 +86,8 @@
 
 .sidebar > .panel.details.css-style > .content > .computed > .details-section.computed-style-variables .scope-bar.computed-style-variables-grouping-mode:hover {
     --scope-bar-background-opacity-override: 0.5;
-}
\ No newline at end of file
+}
+
+.sidebar > .panel.details.css-style > .content > .computed > .details-section.computed-style-variables .computed-style-section {
+    padding-top: 3px;
```

Comment:
You note that this is symmetric with the bottom padding in the commit message - can we make this a variable on `details-section` to guarantee they stay symmetric?